src/main/java/de/geolykt/enchantments_plus/CustomEnchantment.java

```diff
@@ -704,37 +704,37 @@ public T build() {
         private Map.Entry<CustomEnchantment, Integer> getEnchant(String raw, World world) {
             raw = raw.replaceAll("(" + ChatColor.COLOR_CHAR + ".)", "");
             switch (raw.split(" ").length) {
-            case 0: 
-            case 1:
-                return null; // Invalid length
-            case 2:
-                CustomEnchantment ench = Config.get(world).enchantFromString(raw.split(" ")[0]);
-                if (ench == null) {
-                    return null; // Not able to map enchantment
-                }
-                try {
-                    return new AbstractMap.SimpleEntry<CustomEnchantment, Integer>(ench,
-                            Utilities.getNumber(raw.split(" ")[1]));
-                } catch (NumberFormatException expected){
-                    return null; // Invalid roman numeral
-                }
-            case 3:
-                CustomEnchantment ench2 = Config.get(world).enchantFromString(raw.split(" ")[0] +
-                        raw.split(" ")[1]);
-                if (ench2 == null) {
-                    return null; // Not able to map enchantment
-                }
-                try {
-                    return new AbstractMap.SimpleEntry<CustomEnchantment, Integer>(ench2,
-                            Utilities.getNumber(raw.split(" ")[2]));
-                } catch (NumberFormatException expected){
-                    return null; // Invalid roman numeral
-                }
-            default:
-                return null; // Invalid length
+                case 0: 
```

Comment:
This is actually not conventional. The case blocks are "unindented" by one intentation layer from the rest of the block.
I might be fully wrong there though